app/app.py

```diff
@@ -268,6 +268,24 @@ async def matomo_request_log(request: Request, call_next):
     return response
 
 
+@app.exception_handler(RuntimeError)
+async def validation_exception_handler(request: Request, exc: RuntimeError):
+    """
+    Handling RuntimeErrors.
+    Most of the RuntimeErrors are thrown by the siibra-python library when other Services are not responding.
+    To be more resilient and not throw a simple and unplanned HTTP 500 response, this handler will return an HTTP 503
+    status.
+    :param request: Needed but fastapi definition, but not used
+    :param exc: RuntimeError
+    :return: HTTP status 503 with a custom message
+    """
+    logging.warning(str(exc))
+    return JSONResponse(
+        status_code=503,
+        content={'detail': 'This part of the siibra service is temporarily unavailable'},
+    )
```

Comment:
What do we think about adding the `str(exc)` to response so that it's possibly easier to debug on the client side (rather than digging through the logs for the logging.warning?

Reply:
Do you mean as the details part or as a new field to provide more information?

Reply:
I meant as a part of `detail`, but I don't mind as a new key value pair.

when/if we have log aggregation, it might be less relevant, but it would still help those external to us to report the issue to us, IMO